README.md

```diff
@@ -192,18 +194,44 @@ Report usage (using custom template file):
 go-licenses report <package> [package...] --template=<template_file>
 ```
 
+### Save
+
 Save licenses, copyright notices and source code (depending on license type):
 
 ```shell
 go-licenses save <package> [package...] --save_path=<save_path>
 ```
 
+### Check
+
 Checking for forbidden licenses usage:
 
 ```shell
-go-licenses check <package> [package...]
+go-licenses check <package> [package...] 
```

Comment:
Also comment default behavior is disallow forbidden and unknown licenses?

Reply:
Added it to the title